app/src/main/res/layout/fragment_exposures.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <data>
+
+        <import type="org.covidwatch.android.ui.ItemBindings" />
+
+        <import type="android.view.View" />
+
+        <variable
+            name="viewModel"
+            type="org.covidwatch.android.ui.exposures.ExposuresViewModel" />
+    </data>
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical"
+        tools:context="org.covidwatch.android.ui.exposurenotification.ExposureNotificationFragment">
+
+        <ImageView
+            android:id="@+id/btn_close"
+            style="@style/ImageView.BtnClose"
+            android:padding="@dimen/padding_content"
+            tools:ignore="ContentDescription" />
+
+        <TextView
+            style="@style/TextView.ScreenTitle"
+            android:paddingStart="@dimen/padding_content"
+            android:paddingEnd="@dimen/padding_content"
+            android:text="@string/title_exposures" />
+
+        <Switch
+            android:id="@+id/enable_exposure_notification"
+            style="@style/Switch.EnableExposureNotification"
+            android:checked="@{viewModel.exposureNotificationEnabled}"
+            android:paddingStart="@dimen/padding_content"
+            android:paddingEnd="@dimen/padding_content" />
+
+        <TextView
+            style="@style/TextView.ScreenSubtitle"
+            android:paddingStart="@dimen/padding_content"
+            android:paddingEnd="@dimen/padding_content"
+            android:text="@string/exposures_subtitle" />
+
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="50dp"
```

Comment:
use `wrap_content` with `paddingVertical`

Reply:
Yes, totally forgot about this element as I decided to add the underlined logic in the future work =\